bindings/src/Capgemini.PowerApps.SpecFlowBindings/Transformations/UtilityTransformations.cs

```diff
@@ -20,5 +22,23 @@ public static string[] TransformCommaSeparatedStringsToArray(string expression)
         {
             return expression?.Split(new char[] { ',' }, StringSplitOptions.RemoveEmptyEntries).Select(s => s.Trim()).ToArray();
         }
+
+        /// <summary>
+        /// Transforms HumanReadable Integer Expression to numeric values.
+        /// </summary>
+        /// <param name="expression">expression value(1st).</param>
+        /// <returns>numeric value.</returns>
+        [StepArgumentTransformation]
+        public static HumanReadableIntegerExpression TransformHumanReadableIntegerExpression(string expression)
+        {
+            var regEx = new Regex(@"(\d)");
+
+            if (regEx.Match(expression).Success)
+            {
+                return new HumanReadableIntegerExpression(Convert.ToInt32(regEx.Match(expression).Groups[0].Value) - 1);
+            }
+
+            throw new ArgumentException("Unexpected values");
+        }
```

Comment:
I would replace this with something like:

```suggestion
         /// <summary>
        /// Transforms an ordinal number (e.g. 1st, 2nd, 3rd etc.) to a zero-based index.
        /// </summary>
        /// <param name="expression">The ordinal number.</param>
        /// <returns>The zero-based index.</returns>
        [StepArgumentTransformation(@"(\d+(?:(?:st)|(?:nd)|(?:rd)|(?:th)))")]
        public static int TransformOrdinalNumberToZeroBasedIndex(string expression)
        {
            return Convert.ToInt32(Regex.Match(expression, @"\d+").Value) - 1;
        }
```

Reply:
Replaced. That's the perfect.